src/GfGiftAidField.php

```diff
@@ -13,11 +14,78 @@ class GfGiftAidField
 
     public static function run(): void
     {
-        if (! method_exists('GFForms', 'include_addon_framework')) {
+        if (!method_exists('GFForms', 'include_addon_framework')) {
             return;
         }
 
         GFForms::include_addon_framework();
         GFAddOn::register(AddOn::class);
+
+        add_action('gform_field_standard_settings', [static::class, 'addSelectedPriceFieldSetting'], 10, 2);
+    }
+
+    /**
+     * Custom field to enable the user to choose where they would like to pull the total value from.
+     * Add 'donation_total_select' to the array of strings in the get_form_editor_field_settings method of your field.
```

Comment:
`donation_total_select` is old
not even sure this large comment is needed anyway